examples/src/main/proto/greeter.proto

```diff
@@ -29,14 +29,15 @@
 
 syntax = "proto3";
 
-option java_package = "io.grpc.examples";
+package grpc.example.helloworld;
 
-package helloworld;
+option java_multiple_files = true;
```

Comment:
I didn't use the outer class name because the instructions said it should be the name of the file ... but the service should also be the name of the file ... seems these instructions conflict.  If I use the same name as the service the build fails. So I just left it out :)

Perhaps @wora has some guidance?